### ISSUE TITLE
Notification button in top bar have not accessible name

### ISSUE THREAD

**Issue by akhanfir** [Meeds-io/meeds#1296]
**Steps to reproduce**

1. Open stream page

2. Ensure that you have not unread notification

3. Launch a lighthouse report related to accessibility

**Current behaviour**

The notification button in top bar is reported as not accessible.

**Expected behavior**

The notification button in top bar must be accessible, we need to add a aria-label attribute on the button. The value should be "Notifications" and must be i18n.
